app/controllers/application_controller.rb

```diff
@@ -13,11 +13,16 @@ def configure_permitted_parameters
   end
 
   def after_sign_in_path_for(resource)
-    url_for current_object
+    url_for current_user_type unless current_admin
+    rails_admin_url
   end
 
-  def current_object
+  def current_user_type
     user = current_user.user_type
     user.camelize.constantize
   end
+
+  def current_admin
+    current_user.user_type == "admin"
```

Comment:
viết cái gì thế nầy... enum của tui đâuuuuu :(

Reply:
ý a là e nên viết user.user_type = 3 à hay sao a. e chưa hiểu lắm

Reply:
`current_user.admin?`

đọc lại cái enum technique đi xem nào (dam)

Reply:
à hoặc user.admin!

Reply:
current_user.admin? Là đc r toàn

Reply:
ok mọi người :)))